src/components/InventoryTable/EntityTableToolbar.test.js

```diff
@@ -916,21 +910,11 @@ describe('EntityTableToolbar', () => {
 
       await userEvent.click(
         screen.queryByRole('menuitem', {
-          name: 'Export to ' + format.toUpperCase(),
+          name: 'Export to CSV',
         })
       );
 
-      expect(axiosPostMock).toHaveBeenCalledWith('/api/export/v1/exports', {
-        format,
-        name: 'inventory-export',
-        sources: [
-          {
-            application: 'urn:redhat:application:inventory',
-            resource: 'urn:redhat:application:inventory:export:systems',
-            filters: {},
-          },
-        ],
-      });
+      await waitFor(() => expect(axiosPostMock).toHaveBeenCalled());
```

Comment:
Why do we now prefer a test just asserts only the function to be called. Is not it better to have additional assertion on the function call arguments?

Reply:
This assertion is simply only to ensure that clicking triggered the right action. What is passed and how is tested on the hook itself.

Reply:
The test could also just be to assert the export button is available, because most of the implementation for the exporting itself is contained within the hook, and the only thing that the toolbar really does is include or not include the export based on `enableExport`.